### ISSUE TITLE
Volunteer comments double posted

### ISSUE THREAD

**Issue by JonathanNawrocki** [rjenks/Con-Org#115]
Added comment to a user. Jumped to top of page. Hit back button to hopefully return to location. Got the need to refresh page. Refreshed. The comment was added twice.

I know this was a stupid user error, but not sure if anything can be done about it.

**Comment by rjenks** [comment]
Yup.  Hitting back and reposting forms is a common problem.  There are a couple of ways around this, but it's not a trivial change.
